### PR TITLE
Add utopia88 keyboard pcb

### PR DESCRIPTION
**Pull request by Kelwin0313**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kelwin/utopia88/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Utopia88
+
+![Utopia88](https://imgur.com/pRKQIIH.jpeg)
```
Comment (waffle87) [inline keyboards/kelwin/utopia88/readme.md:3]:
```suggestion
![Utopia88](https://i.imgur.com/pRKQIIHh.jpeg)
```

File: keyboards/kelwin/utopia88/info.json
Comment (waffle87) [inline keyboards/kelwin/utopia88/info.json:1]: Please run `qmk format-json` on this file.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/kelwin/utopia88/readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* Hardware Supported: Utopia88 PCBs, atmega32u4
+* Hardware Availability: Discord kelwin4598 or Github(https://github.com/Kelwin0313)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make kelwin\utopia88:default
+
+Flashing example for this keyboard:
+
+    make kelwin\utopia88:default:flash
```
Comment (drashna) [inline keyboards/kelwin/utopia88/readme.md:15]:
```suggestion
    make kelwin/utopia88:default

Flashing example for this keyboard:

    make kelwin/utopia88:default:flash
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/kelwin/utopia88/info.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        "vid": "0x4B53"                                                                                                                                                                                                                                                                                                      
+    },                                                                                                                                                                                                                                                                                                                       
+    "layouts": {                                                                                                                                                                                                                                                                                                             
+        "LAYOUT": {
```
Comment (dunk2k) [inline keyboards/kelwin/utopia88/info.json:28]:
```suggestion
    },
    "community_layouts": ["tkl_f13_ansi_tsangan"],                                                                                                                                                                                                                                                                                                                       
    "layouts": {                                                                                                                                                                                                                                                                                                             
        "LAYOUT_tkl_f13_ansi_tsangan": {                                                                                                                                                                                                                                                                                                      
```

File: keyboards/kelwin/utopia88/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// Copyright 2023 Kelwin
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/kelwin/utopia88/keymaps/default/keymap.c:7]:
```suggestion
	[0] = LAYOUT_tkl_f13_ansi_tsangan(
```

File: keyboards/kelwin/utopia88/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,14 @@
+// Copyright 2023 Kelwin
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/kelwin/utopia88/keymaps/via/keymap.c:7]:
```suggestion
	[0] = LAYOUT_tkl_f13_ansi_tsangan(
```

File: keyboards/kelwin/utopia88/readme.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# Utopia88
+
+![Utopia88](https://i.imgur.com/pRKQIIH.jpeg)
```
Comment (dunk2k) [inline keyboards/kelwin/utopia88/readme.md:3]:
```suggestion
![Utopia88](https://i.imgur.com/pRKQIIHh.jpeg)
```

File: keyboards/kelwin/utopia88/info.json [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+    "processor": "atmega32u4",                                                                                                                                                                                                                                                                                               
+    "url": "",                                                                                                                                                                                                                                                                                                               
+    "usb": {                                                                                                                                                                                                                                                                                                                 
+        "device_version": "0.0.1",
```
Comment (dunk2k) [inline keyboards/kelwin/utopia88/info.json:23]:
```suggestion
        "device_version": "1.0.0",                                                                                                                                                                                                                                                                                           
```

File: keyboards/kelwin/utopia88/info.json [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+            ]                                                                                                                                                                                                                                                                                                                
+        }                                                                                                                                                                                                                                                                                                                    
+    }                                                                                                                                                                                                                                                                                                                        
+}
```
Comment (dunk2k) [inline keyboards/kelwin/utopia88/info.json:120]:
```suggestion
}

```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Pull request closed without merging**